From 2a2d937a495e5170576d46c7b456972898f9cbaf Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 30 Jun 2020 12:02:14 +0200
Subject: [PATCH 1/1] efi_loader: rtc_mktime() called twice

Don't call rtc_mktime() twice with the same argument in
efi_variable_authenticate().

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_variable.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/efi_loader/efi_variable.c b/lib/efi_loader/efi_variable.c
index 6271dbcf41..cd06125f14 100644
--- a/lib/efi_loader/efi_variable.c
+++ b/lib/efi_loader/efi_variable.c
@@ -571,7 +571,7 @@ static efi_status_t efi_variable_authenticate(u16 *variable,
 	}
 
 	/* finished checking */
-	*time = rtc_mktime(&tm);
+	*time = new_time;
 	ret = EFI_SUCCESS;
 
 err:
-- 
2.27.0

